docs/synapse/userguides/storm_ref_lift.rstorm

```diff
@@ -322,450 +405,821 @@ Storm supports a set of extended comparison operators (comparators) for speciali
+- Lift any nodes tagged with those tags (e.g., ``#rep.mandiant.gh0st`` or ``#rep.mandiant.beacon``). This would
+  typically include IOCs such as hashes, FQDNs, IPv4s, URLs, etc.
+- If any nodes tagged with the additional tags (``#rep.mandiant.gh0st``, etc.) are ``syn:tag`` nodes, repeat the
+  process, continuing until no more ``syn:tag`` nodes are lifted.
+- Return the recursively lifted set of nodes (excluding any ``syn:tag`` nodes).
+
+.. NOTE::
+  
+  "Tag the tags" is one approach to provide context to things that tags represent and may be suitable for some
+  use cases. However, the Synapse data model now includes **forms** to represent objects or concepts that are
+  commonly associated with tags, and that can be linked to their associated tag via a ``:tag`` secondary
+  property. For example, a ``risk:tool:software`` node can represent Mandiant's reporting on Gh0st malware,
+  with a ``:tag`` property that could be set to ``rep.mandiant.gh0st``. The node can be used to record
+  additional context about Mandiant's Gh0st, including things like availability, alternate names used in
+  reporting, and so on. In short, using a form with secondary properties to provide "context" (that is still
+  linked to an associated tag) gives you greater flexibility to record that context and simplifies lifting,
+  filtering, and pivoting across similar nodes.
+  
+  See :ref:`analytical-tags-asnodes` for a brief discussion of this concept, or the `User Guide`_ for the
+  `Vertex-Threat-Intel`_ Power-Up (in particular, the `Threat Intel Model`_ section) for additional examples.
+
+
+.. _lift-reverse:
 
-A recursive tag lift performs the following actions:
+"reverse" Keyword
+-----------------
 
-1. For the specified tag (``##<sometag>``), lift the nodes that have that tag (i.e., the equivalent of ``#<sometag>``), including any ``syn:tag`` nodes.
-2. For any lifted ``syn:tag`` nodes, lift all nodes tagged with those tags (including any additional ``syn:tag`` nodes).
-3. Repeat #2 until no more ``syn:tag`` nodes are lifted.
-4. Return the tagged nodes. Note that ``syn:tag`` nodes themselves are **not** returned.
+Synapse indexes property values so that data (nodes) can be lifted (retrieved) and returned quickly. By default,
+lift results are returned in lexical order (i.e., sorted in ascending order), based on the property specified in
+the lift (primary, secondary, universal, or extended) and the way the property is indexed.
+
+The ``reverse`` keyword can be used to return the specified nodes in reverse lexical order (i.e., sorted in descending
+order). To perform a "reverse" lift, specify the ``reverse`` keyword and enclose the lift operation in parentheses.
+
+A "reverse" lift can be followed by additional Storm operations (pivots, filters, commands) just like a "normal" lift.
 
 **Syntax:**
 
-**##** *<tag>*
+**reverse (** *<query>* **)**
```

Comment:
Maybe `<lift>` instead of `<query>`? Only a single lift operation can be enclosed within each `reverse()`

Reply:
Good call - ty!